main.py

```diff
@@ -22,9 +20,11 @@ def new_command():
 
 
 @app.command()
-def if_statements():
+def if_statements(directory_path):
     """Determine number of if statements in a Python directory."""
-    print("In progress")
+    cast_dict = generate_trees(directory_path)
+    if_statements_directory = counter.total_if_statements(cast_dict)
+    print(if_statements_directory)
```

Comment:
Like Maddy mentioned to me, I would add a better print statement here. Just something like `Number of if statements: value`